src/query/execute_statement.c

```diff
@@ -18256,6 +18271,22 @@ pt_check_dblink_password (PARSER_CONTEXT * parser, const char *passwd, char *cip
       return ER_FAILED;
     }
 
+#ifndef NDEBUG
+  if (passwd)
+    {
+      if (strcmp (passwd, "test_check") == 0)
+	{
+	  ciper_func_test (parser, false);
+	  return ER_FAILED;
+	}
+      else if (strcmp (passwd, "test_time") == 0)
+	{
+	  ciper_func_test (parser, true);
+	  return ER_FAILED;
+	}
+    }
+#endif
```

Comment:
This code and ciper_func_test() function don't need PR check.
The code will be deleted after the DBLINK function is complete.